harmoni_core/harmoni_pattern/test/test_sequential.py

```diff
@@ -30,12 +30,13 @@ class TestSequential(unittest.TestCase):
 
     def __init__(self, *args):
         super(TestSequential, self).__init__(*args)
```

Comment:
If you don't define init at all, the super init call will automatically occur.

Reply:
Do you suggest to remove the init in all the testing files for all the packages?

Reply:
Done it